android/build.gradle

```diff
@@ -26,16 +26,20 @@ def getExtOrIntegerDefault(name) {
 }
 
 android {
+  def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION
+  if (agpVersion.tokenize('.')[0].toInteger() >= 7) {
+    namespace "com.sparkfabrik.rnidfaaaid"
+  }
   compileSdkVersion getExtOrIntegerDefault('compileSdkVersion')
   buildToolsVersion getExtOrDefault('buildToolsVersion')
   defaultConfig {
     minSdkVersion getExtOrIntegerDefault('minSdkVersion')
     targetSdkVersion getExtOrIntegerDefault('targetSdkVersion')
     versionCode 1
     versionName "1.0"
-    
+
   }
-  
+
```

Comment:
can you please remove these whitespaces?

Reply:
there are no whitespaces here, just a line break

Reply:
yep, can you remove this change please?

Reply:
I just want the commit to contain only the relevant logic (which is ok)